### PR TITLE
Fix test-amp-ad-network-doubleclick-impl

### LINKED ISSUE #9635: Unit tests in amp-ad-network-doubleclick-impl fail without a dist build
After #9626 is merged, we will no longer be doing a full `gulp dist` for unit tests, and instead, will be doing a `gulp build` followed by a `gulp test --nobuild`. When I tested this change, I noticed that the tests in amp-ad-network-doubleclick-impl fail. It turns out this is due to a missing stub.

See https://travis-ci.org/ampproject/amphtml/jobs/237958534 for full failure logs

### PR DESCRIPTION
**Pull request by rsimha-amp**
Fixes #9635
Unblocks #9626

### CODE REVIEW

**Comment by rsimha-amp** [comment]
Added a comment explaining the change, and added @keithwrightbos and @taymonbeal for review. This PR will unblock a significant gain in speed to the overall PR workflow on Travis.

**Comment by taymonbeal** [comment]
@ampproject/a4a

File: extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js [hunk 1]
```diff
@@ -317,6 +317,13 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
             };
           });
       sandbox.stub(impl, 'getAmpDoc', () => {return document;});
+      // Reproduced from noopMethods in ads/google/a4a/test/test-utils.js,
+      // to fix failures when this is run after 'gulp build', without a 'dist'.
+      sandbox.stub(impl, 'getPageLayoutBox', () => {
+        return {
+          top: 11, left: 12, right: 0, bottom: 0, width: 0, height: 0,
```
Comment (taymonbeal) [inline extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js]: Why these numbers? They look as though they can't be consistent. Unless I've misunderstood?

Comment (rsimha-amp) [inline extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js]: See comment above. These are reproduced from noopMethods in ads/google/a4a/test/test-utils.js, from where they're being picked up during a 'gulp dist', but not during a 'gulp build'.

Comment (taymonbeal) [inline extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js]: Huh, okay. I'll look at them with our team later.

Also, I don't understand how any of this gulp dist stuff works and I think I should, but that shouldn't block this PR.

Comment (rsimha-amp) [inline extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js]: 'gulp dist' does a more comprehensive build, including creating and serving a minified version of the AMP runtime. This shouldn't be necessary for running unit tests, and results in a lot of wasted CPU time on Travis when multiplied across all the commits across all the PRs pushed by people on the AMP team. I'd imagine that you can get your tests to pass without this stub with a little refactoring / reuse of the code in test-utils.js.

Thanks for the review.

**Pull request merged**